src/main/java/io/jawg/osmcontributor/flickr/oauth/OAuthRequest.java

```diff
@@ -58,7 +58,7 @@ public OAuthRequest(String apiKey, String apiKeySecret) {
     /*----------------CODE---------------------*/
     /*=========================================*/
     public void initParam(Map<String, String> params) {
-        this.params.clear();
+        this.params = new TreeMap<>();
```

Comment:
Mmmh makes sense! Didn't know that oauth parameters needed this particular order for signature.
Thanks ;)